lib/peek/controller_helpers.rb

```diff
@@ -9,6 +9,15 @@ module ControllerHelpers
 
     protected
 
+    def append_info_to_payload(payload)
+      super
+      payload[:peek_bar_visible] = peek_bar_visible?
```

Comment:
The only time we should be writing Peek view results into the adapter is when `peek_enabled?` returns true. Right now with the `peek_bar_visible?` guard, it still writes results for every request even if `peek_enabled?` is false.